src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -1650,7 +1666,9 @@ internal IEnumerable<Cci.SecurityAttribute> GetSecurityAttributes()
 
                         if (attribute != null)
                         {
+#nullable disable // Can 'constantValue' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
With our standard library? No. With some custom standard library where this type has no constant value? Yes.

Reply:
I'm planning to leave this for a follow-up